src/server/qtquick/woutputviewport.cpp

```diff
@@ -129,6 +130,14 @@ QSGTextureProvider *WOutputViewport::textureProvider() const
     return d->bufferRenderer->textureProvider();
 }
 
+WBufferTextureProvider *WOutputViewport::wTextureProvider() const
+{
+    W_DC(WOutputViewport);
+    if (QQuickItem::textureProvider())
+        return nullptr;
```

Comment:
Why

Reply:
Let wtexture provider be consistent with texture provider, but I'm not sure if this is necessary.

Reply:
No